### ISSUE TITLE
E c o n o m y

### ISSUE THREAD

**Issue by nuklearcellphoneg** [d3athrow/vgstation13#10837]
There are very few items that have value in this game. For example:

We are in space, equipment for dealing with that should hold value. Every character comes with a mask and air tank. The hallways are littered with O2 closets. There are several locations with large tank dispensers.

Paramedics get a hardsuit. There are quite a few hardsuits in EVA. Captain gets his own hardsuit. Security has separate hardsuits in the armory. IIRC firesuits provide some hardsuit level protection.

Atmos hardsuits have much more value, because they protect against plasma fire.

We have antagonists, so weapons are a necessity. In particular are guns, as they allow for distance which make a variety of items, stun prod, stun baton, stun gloves, useless.

Guns are present in the armory. Guns can be bought by Cargo for cash, which itself has little to no value, in particular with with the $5,000 cargo has access to, as well as the larger cache the HoP and Captain can get. Guns can be produced by Science. Traitors can purchase a revolver as well as other items, such as the crossbow, arguably a much bigger value than the revolver(regens ammo, causes a guaranteed stun). 

To a chemist, larger beakers hold value. They take time to procure which takes away from their job. The large beakers themselves are not difficult to create or carry around. Bringing larger beakers to a chemist tends to make them very open to making you something. To Science, minerals hold value. Science and Mining have a great bartering relationship. Miners bring in the Minerals, and Science gives them all kinds of shit. Why?

It takes time away from doing Science to do the Mining. Time is very valuable in this game. 

The Chef largely relies on the Botanist for ingredients. The bartender relies on a couple of departments if they want specialty drinks. Cargo relies on the QM's money(and by proxy mining).

---

The basic take away and discussion I'd like to result from this is, first and foremost we've got to give items more value. If a bartering system can't exist, a working money-based economy also cannot. Yes, you can barter as things stand- however, if you're not willing to barter, there typically exists an alternative and relatively convenient way to get what you want.

---

The above is all kind of essentially fact, or the way things are, somewhat subjectively. The following is my personal suggestion to how things could be directed towards having an economy.

1) Atmos hardsuits are valuable. This is because they are rare compared to most hardsuits. To increase value, decreasing the presence of a large amount of goods to skeletal levels is our best bet. We should come up with a list of things which are very readily available which we could decrease the availability of. To use the example of the Titanic, we should have less life boats than we have people(starting out).

2) A lot of things can be created in various departments. Certain things can only be brought station side through cargo. Right now that's mostly a few specialty items with no real utility. If a lot of things can only be obtained through Cargo(air tanks, masks, even the box in your backpack), this gives much more incentive to ordering via Cargo, which would give more incentive for people ordering to pay for their items, or for Cargo to have, say, air tanks ready for sale. 
- There is a request console in various departments, for example in Chemistry. Cargo could use a big update to its UI for the consoles as well as the way we request items between departments, to allow for cash transactions, to fill out an order for Cargo from your department, etc etc.
- If Security wants a pill bottle of 14 15u hyperzine pills, let them remotely pay and order this from the Security department.
- If the Bar is very dirty, they can pay the Janitor a nominal fee to come by for a bit.
- If Medbay wants their equipment upgraded, they can pay Mechanics to do so.
- If an assistant wants to make a mini medbay, they can request science to build them a Cryo.

---

The main things I think we need to consider(and order of importance to consider) are:
- Increase the value of items via rarity and ability to produce them
- Revamp Cargo console's UIs
- Revamp the system of requesting goods
- Consider removing the majority of the cash readily available except where necessary: Give Cargo a minimal fund. Give Science a minimal fund to be able to buy Minerals from Cargo.
- Once the above things are done, consider THEN to assign a cash value to things: Revamp the costs of Cargo items to accommodate this. Consider the most requested size of a Hyperzine pill to be 15u: add a standard cash value to this. Consider the mineral cost of building certain things and the research required, like a Cloner, in order to assign a cash value for buying minerals and for selling the construction of a cloner.

**Comment by Kurfursten** [comment]
I think economy is possible, thanks to Deity's PDA printing. Since I started playing Trader, I have personally taught more than 30 people that you can print money at your PDA. It's a matter of learning how easy it is for most people. We need to remember that a working economy is not just a code issue, but an education issue.

Maybe you could run an event with a reward for whoever makes the most successful money via trading or something?

Don't forget to reconsider balancing the income of miners and thus lowering the cost at their vendor as well.

**Comment by pvak** [comment]
make cargo be able to sell stuff to centcomm

**Comment by ConkerDev** [comment]
Space Station 13 is not a good game to use in game currency. The reason is that the gamemodes we have assume the crew is cooperative. In that sense, SS13 is a COOPERATION game, and that can be seen in game. The more cooperative a crew is, the best they perform against the odds. This is also why Nations, Berlin, and Taxi are hated; they artificially difficult intradepartamental cooperation. This can also be seen in the random dificulties the crew faces:
-Malf AI keeps the crew constantly in fear of silicons, making their relations always somewhat tense
-Traitors make the crew never fully trust anyone else.
But not everything has to be based in fear.
-Communications is a department (and therefore subject to mishaps) preciselly because the radios are the spessman's strongest weapon. Without them, Condoms can't call for sec, nor Hunter McValids can announce there's aliens in the AI chamber.

Remember that this game is Paranoia with atmospherics. That's the game's theme, fearful cooperation. Economy has always gotten in the way of cooperation, and it doesn't fit at all with the theme of the game (nor it would make any sense for a station to charge its employees for materials required for their job).

But we can, as @nuklearcellphoneg suggests, make items more valuable in other sense. We can make them scarce. In a game where the always far away long term ideal is to make any part of the station buildable, scarcity isn't given by how much something costs, or how many of them there are, but how hard is it to make. Phazon mechs are very expensive because of the high ammount of cooperation and expertise required to 
1. Callibrate correctly a teleporter
2. Send a miner to a location outside of telecomms range
3. Mine the materials
4. Send the miner back
5. Smelt the materials
6. Bring them back
7. Build a mech

This usually may take at least half an hour of extra time (after main research has been completed), and it will require at least 3 people cooperating the whole time, spending time doing this instead of atending other requests.

So, yes to making some items more scarce as a balance. No to economy

tl;dr: We already have an economy, a time economy.

**Comment by derpycobra** [comment]
> be sec officer / med doctor
> hey cargo we need more hardsuits/medical stuff/weapons/whatever
> it will be 300$
> but I don't have 300$
> well fuck you then

Does that mean I will now get paid before I arrest someone? And do I need to pay a doctor everytime I need to get treated in medbay?
There is a problem with certain jobs, mainly security, command, engineering and medical, where you don't get paid for the work you've done. How would you solve that?

**Comment by jknpj** [comment]
Maybe a random event where centcomm ask for specific things and the crew can make make big profit if they're able to send it <s>or else they get a paycut/bluespace artillery</s>

**Comment by ComicIronic** [comment]
I would fully support paying officers for arrests a la Libertarian Paradise.

**Comment by Kurfursten** [comment]
> Officer, greyshirts broke my window
> I'll track him down and arrest him if you pay me $25

**Comment by jknpj** [comment]
Maybe centcomm could pay them for every confirmed antagonist <s>eliminated</s>briged

**Comment by PJB3005** [comment]
Let's not encourage validhunting.

**Comment by Kurfursten** [comment]
There's also the weird kind of brig test that provides

> We'll know he's valid if Centcomm pays us when we brig him

**Comment by icantthinkofanameritenow** [comment]
Nuke, you also forgot the ghetto hardsuits that can be made with firesuits, space blankets, and cable (and their matching helmets, made from fire helmets, gas masks, space blankets, and cable). Out of that, the only one of those items that can be called "hard to find" is the space blanket, and even those can be found easily enough in the plain white medkits. 

If this is going to go anywhere, it would be wise to start by removing ghetto hardsuits. While they provide no protection from radiation or viruses and are absolutely worthless as armor with a protection of 0 in every category, they're still spaceworthy and easy enough to get. In their place, having a crate with generic "space suits" like the one that starts in Telescience might be a good replacement.

**Comment by Spacedong** [comment]
no, why would crew members charge each other for shit, they're all there to cooperate on the mission nanotrasen told them to perform, sharing shit and working efficiently

**Comment by Intigracy** [comment]
Let's make all vending machines require spending money to get things from them, except premium items needing just a coin.

I'm heavily against miner income being nerfed because being a miner is incredibly boring and repetitive, at least they get the benefit of getting mad dosh from doing mad work.

Using the autolathe could cost money on top of materials (I'm talking flat rates here).

Printing things from the protolathe could cost money on top of materials (I'm talking flat rates here).

**Comment by clusterfack** [comment]
Everyone is heavily against something which is why nothing will EVER get done on this topic.

**Comment by SangriaMonkey** [comment]
> - work for the same company
> - have to pay your coworkers to cooperate

:eggplant:

**Comment by Intigracy** [comment]
https://en.wikipedia.org/wiki/Company_town

**Comment by SangriaMonkey** [comment]
But they're not even separate enterprises. It's all under one roof.

**Comment by Intigracy** [comment]
https://en.wikipedia.org/wiki/Company_town

It's literally a corporation in charge of every aspect of living in the town. Businesses, housing, everything. You spend your money and it goes right back to them.

Kinda like on SS13.

**Comment by SangriaMonkey** [comment]
Office buildings also have different departments for different workers and believe it or not, they don't function like Ole Timey Coal Towns

**Comment by ComicIronic** [comment]
Yeah, literally scrip.

**Comment by nuklearcellphoneg** [comment]
@ConkerDev The thing I definitely most resonate with is

> tl;dr: We already have an economy, a time economy.

That is most definitely true: Anytime a transaction takes place(not necessarily a cash transaction) it largely depends on the time it takes away from doing your job in order to obtain an item.

Let me reiterate, what I want more than anything is a decrease of availability of a lot of items(hard suit, O2 tanks) in order to facilitate bartering. 

@derpycobra Consider that once we can have bartering exist properly, then you can reduce the availability of $$$. They may not necessarily pay you to catch a criminal, but they may A) Reward you(you come in and stun cuff someone murdering medbay, CMO gives you some cash) or B) Medbay knows there's a cult and pays you to patrol them.

First we need to better facilitate bartering and by proxy the value of items.

@jknpj Again, not too worried about a cash economy yet: We should focus on making bartering viable for now(which truly, it is not)

---

Let me reiterate: At this point, a Cash Economy is useless and can only function through RP. What we need first is bartering. Consider:

The farmer grows the wheat.

The blacksmith makes the metalworks.

The soldiers protect the people.

The King leads the people.

Every Job needs the wheat for food.

---

 The farmer needs tools for farming, and so trades to the blacksmith. This is a rare transaction: Every time the blacksmith needs to eat, the farmer does not need new tools. He does need them upgraded, maintained but again not as often as the blacksmith needs to eat.

The soldiers need tools to fight. The blacksmith gives them the weapons for protection, the farmers gives them wheat for protection. 

There is then the question of the King: The people can live without him, so what necessitates his position, outside of leading the 'people'(let's be honest, soldiers)?

The King gets most of the wheat: The wheat is needed by the smith, the soldiers, the king, and the farmer. 

---

The farmer gets a pittance of his own wheat, but is allotted tools and protection.

The blacksmith is given wheat and protection.

The soldiers get a portion of wheat, as well as weapons.

The King controls the economy. They distribute the food, which is what facilitates the bartering system. What causes the bartering system to fail is this:

Everyone needs the wheat, but not everyone always needs the protection, the tools, or honestly, the King. This is then where a monetary system is established, which also empowers the king.

Now then:

@Spacedong 

> no, why would crew members charge each other for shit, they're all there to cooperate on the mission nanotrasen told them to perform, sharing shit and working efficiently

Sometimes as Chemist I go to the bartender and ask for orange juice, lime juice, tomato juice, and milk cream in order to make DD.

This is easy to give me, it benefits them, they have plenty of those juices to last them awhile. They don't give it to me. They won't give it to me. I go to Cargo for similar reasons, to get large beakers: They straight up refuse. I go to the Botanist to ask for honey. They have no plans make any, plus Cargo refuses to process their order.

I have the best chance to:

Get the botanist to make my honey. The botanist will probably want my Unstable Mutagen.

2nd best chance:

Get the ingredients from the barkeep; he has a slim chance of maybe wanting ingredients I can give him(m a y b e).

Then however there is no chance with Cargo. If they don't really feel like making my stuff, I have no incentive from my job to get them to make the stuff.

This is the point where Cash comes into play OR

A commodity acting as a bartering item:

So, what could act as a bartering item for us in SS13?

**Comment by ComicIronic** [comment]
I think coins would be a good start. Giving every player a way to get coins easily - possibly through some kind of auto-mint that miners would have to fill with mats - in exchange for cash would be a good start. Then players could move coins around, which is easier to handle to cash, and has a more direct use in premium vending.

**Comment by Kurfursten** [comment]
@ComicIronic We have one of those, Getmore Chocolate Corp

Choco coins are fully usable as coins

**Comment by ConkerDev** [comment]
Wonder why no one uses them

**Comment by FudgePucker** [comment]
I think cargo is the biggest problem here.
Using your money right now outside of the food venders is dependent on them not being fucking retards. And cargo can get enough cash to not have to do their job for the rest of the station.

Having a intergalactic blue space mail system that is dependent outside of cargo could help things out. Want a carm. projector? You can buy one, at a very high price. By adding items that players will possibly want will add real value to the cash. But you will need to nerf departmental money and miners though, if they can get enough cash to buy out everything then cash goes back to having little value because cargo can get you what you want, some of the time.

Bartender wants to order a bottle of nothing, cargo is filled with retards, he can just charge for some of his drinks and buy one. Chemist wants a blue space beaker, science is filled with retards, he can charge for bottles of hyperzine to buy one.

This comes back to what Conker said, we have a time economy.
If we allow people to buy items with out depending on shitter city- I mean, science and cargo, that will help speed up the process of their jobs, then they will start valuing credits at a much higher value.

By adding a new ability to buy objects outside of venders, then people will be more likely to value credits because dealing with cargo and their shuttle fucking sucks.

Cargo will still have value in the round because they can order things in bulk and not have to deal with a massive surcharge.

The reason no one gives a shit about economy is because there is very little use for credits outside of PvE, so pretty much charging for food makes you a fucking giant faggot, because you have zero use for credits. By adding another use for credits you can charge for food and not seem like a giant fucking asshole because you want to buy some special ingredients.

TL;DR: Fuck cargo, let players buy shit without them.

**Comment by icantthinkofanameritenow** [comment]
Actually, just remove Cargo completely or make their own account dependent on people paying them. If they want their gun and.glove crates, they ought to pay for it themselves. Only the AI and captain should be able to use the station account for money.

**Comment by ConkerDev** [comment]
> people want to get rid of cargo to implement a non fun feature

No. SS13 is not a company town because it isnt the spessmen's home (they are there for a shift to get shit done and get back). Its similar in that sense to an oil rig.
Nor it makes sense to pay for tools for your job. "Hey I need metal to fix the solars that the meteors broke" "tough luck buddy it will be 300 pomfcoins"
Nor is it fun. Test it. Become the QM, and instruct the cargo shitters to ask for money for every single transaction. Enjoy how fast the crew ignores cargo completelly and goes to science mining.

The gun and glove shit is part of the fun in cargo, for fucks sake. What long term projecs would cargo have apart from becoming independent and building Inner Heaven?

**Comment by icantthinkofanameritenow** [comment]
The thing is that in practice the "guns and gloves" shit means they're too self-absorbed to be there when they're needed, which isn't fun for most other people involved. A balance has to be struck.

**Comment by ConkerDev** [comment]
A roboticist can do that too. So can science, mining, genetics, engineering, atmos...
The usual response is "You fucking shitheads do your fucking job"
Maybe a job ban too. Do the same with Cargo. All fun and games, but you have to actually listen and do your requests. If no one controls cargo, then of course they aren't going to do shit

**Comment by nuklearcellphoneg** [comment]
Again,

We should focus on increasing the value of items already existing in game before we even consider moving onto money. Bartering rarely works because items are plentiful and can be found in many places as pointed out many times above.

**Comment by icantthinkofanameritenow** [comment]
How so?

 I guess the easy way would simply be to limit Cargo's ability to order the items you want to make more valuable. Not just as in price and access - I'm thinking something like a flat quota to how many of a given crate they can order. Any more than that amount, and they have to make a special order to CentComm explaining what they need it for (or at least have to pay at least twice the normal amount of money to order it. 

But still, that may not work given that miners can make more money than the rest of the station combined and make any such price hurdle irrelevant in an instant. There's got to be better ways to create scarcity than that.

**Comment by jknpj** [comment]
There's an issue with multiple forms of acquiring items and "guetto" items as well. An emag and the mechanics' reverse engine(THAT HAS NO ACCESS RESTRICTION) can even make Spur a common item.

**Comment by FudgePucker** [comment]
Don't forget that making some items scarce would hurt engineers. They don't have a way to sell goods.

**Comment by nuklearcellphoneg** [comment]
@icantthinkofanameritenow  Well, personally I feel that:

1) Remove boxes, bandages, air tanks, air masks, AND back packs from round start spawn.

2) Remove most of the Air Lockers; make their placement more strategic(maint, specific rooms, less life boats than people)

3) Hardsuits: Engineering needs them, Mining needs them. Security needs less. Paramedic needs one.

4) Consider what people NEED from Cargo: Guns, beekeeping, metal, glass, autolathe stuff. Consider what people abuse Cargo for: Guns, Spec Ops crate(Comes with an incendiary grenade, box of emps, box of smoke bombs, and a sleepy pen for the cost of one emag and 10 space bucks a piece)

5) Reduce equipment per department: Engineers spawn with tools, there are lots of tools in the assistants' two rooms: Bunch of tools in engineering. Cargo can order/create tools. Science can create tools. Mechanic can create tools. 

@FudgePucker 

> Don't forget that making some items scarce would hurt engineers. They don't have a way to sell goods.

You can't very well charge for making repairs but it could be plausible to charge for a power bill somehow, and there's also the ability to charge for upgrades and renovations. Want a room added on? Want to expand cargo? Pay an engineer to do it.

Here's a list of feasible things people could eventually charge for(KEEP IN MIND: At this point money is just fluff, consider how, if at all, bartering works in these regards as things stand):

1) Medbay: Medicine, medical supplies, special attention to tracking beacons, genetic backup, etc

2) Security: Selling their supplies, Rent-A-Cop activities, excess officers? Pay one to call medbay home. 

3) Engineering: Upgrades and renovations

4) Atmos: Upgrades and renovations, properly mixed air canisters, anti plasma fire canisters, etc etc

5) Science: Upgrades, new equipment, special research items, grenades(metal foam to seal breaches, sleepy grenades for sec, etc etc)

6) Chef: Food, Specialty food, nutriment for Medbay(F O O D, then use that mad dosh to pay a Sec Officer to protect you since people are gonna dunk you for not giving away your food for free :^) )

7) Bar: Specialty drinks, supplies, shaker, flask

8) Botany: Specialty ingredients, honey, specialty plants(medbay super healing kudzu in the back patient room??)

**Comment by nuklearcellphoneg** [comment]
First and foremost if we ever want to move towards cash-centric economy, we need to foster a bartering environment.

An alternative form of cash economy we could work towards would be the idea of departmental currency. In this the 'a rose by any other name would smell as sweet' applies, doesn't matter what you call it but this is an idea I had.

First, have an extremely secure vending machine for each department: Cargo, Medbay, Security, etc, with goods for sell. If, say, a mechanic or engineer uses it, they could list out: 'Build Chem Dispenser $1,000' and then it pops out a Mechanic Chip with the task on it, and you hand it over and they do the task and they can redeem it for $1,000 or whatever. That's one part of it.

Another is the idea of Department Vouchers, where the Rose by any other name... bit came from. Consider it like this:

You have Space Credits, the main currency.

Then each department has Vouchers. Medbay, Cargo, Engineer. Each member of each department has some vouchers, let's say three or five or so apiece, with each set bound to their name once activated. So it's like, Medbay Voucher, then I do something with my PDA and it becomes 'Medbay: Bob Dylan Chemist Voucher'.

So, Chemistry: Round start, a Cargo dude comes by and gives you two large beakers for no reason. You're busy, you don't need anything, you're appreciative, but you don't have anything to give them. So you give them a Voucher: Bob Dylan Chemist Medbay Voucher. That says to everyone, you did this dude a favor, and can act as a currency of sorts. 

So, a Cargo Tech goes by Medbay: Gives chemistry some large beakers, changes the battery for the APC in genetics, gives the CMO: You know have, say, a CMO MB Voucher, Geneticist MB Voucher, and a Chemistry MB Voucher.

Let's say you want to come by, get a genetic back up made, fill up a bottle of medicine. On the one hand, you could pay cash for it, but you have three Medbay Vouchers; you've done three big favors for them, now it's time for your payback.

**Comment by icantthinkofanameritenow** [comment]
> 1) Remove boxes, bandages, air tanks, air masks, AND back packs from round start spawn.

That might be more than a little bit extreme, IMO. Even then, you should remember most jobs have a locker where their corresponding backpack spawns, as well as the generic backpacks in the wardrobe room across from tool storage. 

Also, you should share this idea in the forums, ingame, or (if you feel up for it) the thread. We need all the outside opinions we can get or it'll all be violently rejected.

**Comment by FudgePucker** [comment]
> 1)

Hahaha no. Backpacks are very important, like really, very important, first thing you are going to have to buy is a backpack. Making them scarce is just stupid. Also removing bandages is stupid because they got a round start one because bleeding damage is OP and if you don't get a bandage on your wound fast enough you bleed out to death. Also removing airtank and mask round start would make welderbombing round start EVEN MORE ANNOYING.

> (2

See above

> (3

Make public EVA have shitty gray suits and give more hardsuits to departments. Now if you don't want a shitty space suit you are going to have to do some under the table deals with a department.

> (5

Like I said before, making necessities required to buy at round start is just fucking annoying. Want to fix a window? Don't want to wait on engineers to get there 10 hours later? Time to wait in line at cargo while they are jerking each other off for a screwdriver and crowbar.

> The rest of that reply

You are basing your theory on the hopes that people will interact between departments.
That never happens.

**Comment by icantthinkofanameritenow** [comment]
Unfortunately, FudgePucker is right- getting departments to interact is almost always an exercise in futility. Plus,w hen people need something on this server, they're more likely to break in and take it themselves than ask for it. 

 I think the time to use the carrot is long past- it's time to bring out the stick.

**Comment by ComicIronic** [comment]
We can encourage department interaction by building a good economy system, and we all know what Fudgepucker has to say about interaction already.

**Comment by icantthinkofanameritenow** [comment]
@ComicIronic

Let me.guess- "over my dead body"?

**Comment by ComicIronic** [comment]
#10787

**Comment by FudgePucker** [comment]
I think that's why my idea of blue space ordering would be best. People will pay and charge people just to not have to deal with other people.

**Comment by nuklearcellphoneg** [comment]
In order to build a good economy system, we first have to be able to see a successful bartering system of sorts.

Anyway, you see a ton of interaction already:

Mining-Science is the best example

Chemistry-Botany

Chemistry-Bartender

Mining-Bartender

Cargo-Assistants

Security-Science

Security-Cargo

Engineer-Cargo

**Comment by FudgePucker** [comment]
More uses for bartender drinks.
More powerful meds from plant mutations.
Two ideas that would provide more uses for service.

**Comment by FudgePucker** [comment]
I still hold a firm belief that people will pay to not have to deal with other people.

One idea for blue space mail is an autosurgry machine. One time use and let's you perform any surgury on your self. After use it deconstructs into a pile of metal. Don't want to ride the wild ride? Got 1000 credits to burn? Just pay up to not deal with that shit. Want to implant an object into your self with out getting snitched on? Just have the bot do it.

Also add new unique items to departments not available to any other department (science and cargo get everything)
One idea is heirloom seeds for botany.
Weed, contains THC, slows down metabolism rates for chemicals, makes you hungrier faster, slurs your speech, makes you giggle, and 'soften' the graphics while having a sharp outline on objects (messing with depth perception)

**Comment by icantthinkofanameritenow** [comment]
@FudgePucker

Your autosurgery already exists. It's called "a medical cyborg and law 2-ing it not to snitch". 

And how dense are you to not know that ambrosia was always just weed with a different name?

**Comment by FudgePucker** [comment]
Medical cyborg will be scratching at that command all round to get you back. He will want to valid you so hard in that situation that the second your command gets overwritten by a separate command they will spill the beans on anything else you do. They know that you are guilty of everything that is accused of you. This isn't valid hunting, this is them getting you back for manipulating them. Medical cyborgs have to do some surgery on you involves a long process of moving around tools. So now they know that you have a max cap bomb in your chest. It would be pretty shitty of a fucking cyborg that needs to take care of the crew because of law one just tries no way to stop you from blowing up the crew. 'oh goodness me, if I try to stop the guy with a bomb in his chest I will be shitposted at for being a shitter for trying to stop half the crew from dying.' You have now involved your round into his round, so going after you isn't validhunting.
Also what if another antag has hacked the cyborg? Letting him put you under while not under asimov can lead to your death.
People will pay to not have to deal with other players.

When I smoke half a joint of weed my liver doesn't start to imminently fail and cause me to die.
Oh but yea ambrosia is totally like weed.

**Comment by icantthinkofanameritenow** [comment]
I can solve your asinine hypothetical scenario in three steps. 
1. Get compressed matter implant.
2. Use implant on bomb, then on self.
3. Spawn bomb, blow self up. 

Other ideas: Explosive implant, emagged mediborg, emagged MoMMI with surgeon's switchtool.

Wow, that was hard. 

And what is it with you and trying to erase every possible consequence for your actions? (Also, for the record Ambrosia was even CALLED weed before it got a name change for some reason. So stop whining about that. )

**Comment by ComicIronic** [comment]
> don't want to interact with other players? shell out for a machine to bypass them! 

Fudge, are you a meme

**Comment by nuklearcellphoneg** [comment]
Science - Mining is like, the perfect interactive experience, Cargo is kind of involved in this but not really tremendously. Let's look at a different relationship:

Botany - Chef - Bartender.

Science will give Mining many goodies, only in exchange for minerals, very very readily.

The Chef, however, can't really get Botany to make him foodstuffs: Arguably, the chef has very little he can offer the Botanists as a reward.

**Comment by FudgePucker** [comment]
You guys are suggesting we beat players with a stick until they start having an economy.
What I am suggesting is that we have one really nice giant carrot. but only one of them, and all the crew has to fight for part of it.
They already get crates filled with shitty carrots, all of crates have a different flavor. Sure you can go around and try to get some different flavors, but then you have to deal with other people and hope they are willing to trade.  They might be content with their shitty carrot. This one carrot has such deep flavors, you don't have to try to go and get those shitty other carrots.

So now people will be in competition for a part of this carrot. They will trade off their shitty carrots for a part of the really nice one.

@ComicIronic I made the tier three parts requiring only metal and glass and the brawndo acts as adminzine for plants PRs, what do you think :^)

@nuklearcellphoneg The main problem with botany chef and bartender is that  they are one department, but they act as if they are separate. If the scientist in research wants to become a slime person, they can go down to xenobio and see if he has a slime core ready. If the scientist in xenobio wants a temp gun, he can see if research has gotten done with development. They all work as one department instead of telling each other to piss off. HoP should get involved more with the department, as he is the head of them. If the HoP is running around looking to arrest people, but hasn't stopped by and make sure that the kitchen and bar are running smoothly with no internal conflict, then they deserve to be job banned for being a bad head.

**Comment by icantthinkofanameritenow** [comment]
The problem there is that your carrots are too nice to ever be included in the first place, and would effectively replace any form of interaction between actual players. Seriously, buyable instant risk-free surgery, no questions asked? 

And far more damningly to your ideas is the fact that money is still just drip-fed to everyone even if they've been braindead all round. It effectively defeats the purpose of even having an economy at all. This is a space station with paid workers, not a containment pen for a bunch of trust fund brats.  Why give them that really nice carrot when they can get it without ever lifting a finger?
